src/pair.c

```diff
@@ -551,111 +551,156 @@ int get_pair_osc_arg_index(char* varname, char* oscargs, uint8_t argc, uint8_t s
     return i;
 }
 
```

Comment:
is_ws() checks whether the string argument contains nothing but whitespace. I'll add a comment there.